app/utils/serializer.scala

```diff
@@ -161,8 +159,8 @@ package object serializer {
         "start" -> c.startDate,
         "end" -> c.endDate,
         "deadline" -> c.deadline,
-        "logo" -> c.logo,
-        "thumbnail" -> c.thumbnail,
+        "imageUrls" -> asScalaSet(c.imageUrls),
```

Comment:
A major problem arose when I was testing the deployment with postgres: in contrast to the h2 database, it does not seem, that postgres supports saving and loading sets for `imageUrls` and `infoTexts` as it is currently implemented.